### PR TITLE
Added commands to list vpx contents and script.

### PR DESCRIPTION
**Pull request by MarijnStevens**
./vpxtools ls <table> list the files inside an vpx without extracting.

### CODE REVIEW

File: src/vpx/expanded.rs [hunk 1]
```diff
@@ -50,6 +50,141 @@ pub fn extract(vpx_file_path: &Path, expanded_path: &Path) -> std::io::Result<()
...
+    // -extract_info
+    files.push(
+        root_dir_path
+            .join("TableInfo.json")
+            .to_string_lossy()
+            .to_string(),
+    );
+    // TODO -extract_gameitems
+
+    for file_path in files {
```
Comment (francisdb) [inline src/vpx/expanded.rs:181]: can you keep console io in main?

So returning the files here and println in main, that way we can test this function

File: src/vpx/mod.rs [hunk 2]
```diff
@@ -203,6 +203,17 @@ pub fn verify(vpx_file_path: &PathBuf) -> VerifyResult {
...
 
+pub fn cat_script(vpx_file_path: &PathBuf) {
+
+    let mut comp = cfb::open(vpx_file_path).unwrap();
+    let version = version::read_version(&mut comp).unwrap();
+    let gamedata = read_gamedata(&mut comp, &version).unwrap();
+
+    let script = &gamedata.code.string;
+
+    println!("{}", script);
```
Comment (francisdb) [inline src/vpx/mod.rs:214]: Same here, returning the script and printing in main

**Review by francisdb — CHANGES_REQUESTED** [review]
minor improvements

**Comment by francisdb** [comment]
and looks like `rust fmt` was not applied

**Comment by MarijnStevens** [comment]
Updated commands to println in main.rs

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
```
Comment (francisdb) [inline .vscode/launch.json:1]: can this be removed please?

**Review by francisdb — APPROVED** [review]

**Pull request merged**